app/views/entries.scala.html

```diff
@@ -23,20 +23,26 @@ <h4 class="subheader"><small>Total entries</small></h4>
                     <tr>
                         <th>hash</th>
                         @fields.map { field =>
-                        <th>
-                            <span class="field-value">@controllers.html.Utils.toLink(field)</span>
-                        </th>
-                        }
+                            @if(controllers.html.Utils.isDisplayField(field.getName, controllers.App.instance.register.name)){
+                                <th>
+                                    <span class="field-value">@controllers.html.Utils.toLink(field)</span>
+                                </th>
+                                }
+                            }
+                        <th></th>
                     </tr>
                     </thead>
                     <tbody>
                     @records.map { record =>
                     <tr>
-                        <td class="hash"><span class="field-name">hash</span><span class="field-value"><a href="/hash/@record.getHash">@record.getHash.take(7)</a></span></td>
+                        <td class="hash"><span class="field-name">hash</span><span class="field-value"><a href="/hash/@record.getHash">@record.getHash</a></span></td>
```

Comment:
did you really want to display the full hash rather than just the first 7 chars?